readme.md

```diff
@@ -150,8 +150,10 @@ Stream combining/interleaving [`stdout`](https://nodejs.org/api/child_process.ht
 
 Run a file through a forked process. The default sub-process is the current (node).
 
-It introduce a communication channel (IPC) that allows messages to be passed back and forth between the parent and child.<br>
-Note that the `shell` option will be ignored.
+Same as `execa('node', [file, ...arguments], options)` except (like [`child_process#fork()`](https://nodejs.org/api/child_process.html#child_process_child_process_fork_modulepath_args_options)):
+  - the `execPath`, `execArgv` and `silent` options can be used
+  - the [`shell`](#shell) option cannot be used 
+  - the [`stdio`](#stdio)-related options defaults to `inherit` instead of `pipe`
```

Comment:
We should also add:

```
- an extra channel [`ipc`](https://nodejs.org/api/child_process.html#child_process_options_stdio) is passed to [`stdio`](#stdio)
```

Also the first two sentences (`Run a file... current (node)`) and the last sentence (`Returns the same object`) can be removed as they are implied by those new sentences.

Reply:
For the last sentence I agree, but I think it's nice to have a little "prosaic" description of the function as introduction.

Reply:
Sure about the first sentence. However I find the second sentence `The default sub-process is the current (node).` a little confusing. Also the next sentence `Same as execa('node', ...)` should already indicate that the default child process is `node`.

Reply:
Alright !